src/patterns/data/default/entity_attribute.tsv

```diff
@@ -4100,3 +4100,177 @@ OBA:2050052	occipital lobe volume 	UBERON:0002021	occipital lobe	PATO:0000918	vo
 OBA:2050053	lentiform nucleus volume	UBERON:0002263	lentiform nucleus	PATO:0000918	volume
 OBA:2050097	serum amyloid A protein measurement	CHEBI:138152	serum amyloid A	PATO:0000070	amount
 OBA:2050098	serum paraoxonase/arylesterase 1 measurement	PR:000013021	serum paraoxonase/arylesterase 1	PATO:0000070	amount
+OBA:2060000		NBO:0000112	aggressive behavior towards inanimate objects	PATO:0000161	rate
+OBA:2060001		NBO:0000223	contextual conditioning behavior	PATO:0000161	rate
+OBA:2060002		NBO:0000339	motor coordination	PATO:0000161	rate
```

Comment:
I wonder about this one @rays22.. 

`motor coordination` is a trait IMO. `rate` of `motor coordination` feels a bit off.. Its not really a behaviour, is an ability.

Reply:
I think it is a valid interpretation of 'motor coordination' to mean a process as defined in NBO.
To me, a quality of 'motor coordination' looks more like a trait.